tests/manage/mcg/test_bucket_creation_deletion.py

```diff
@@ -148,14 +148,37 @@ class TestBucketCreation(MCGTest):
             "1-CLI-PVPOOL",
         ],
     )
-    def test_bucket_creation(
-        self, bucket_class_factory, bucket_factory, amount, interface, bucketclass_dict
+    def test_bucket_creation_deletion(
+        self,
+        verify_rgw_restart_count,
+        mcg_obj,
+        bucket_class_factory,
+        bucket_factory,
+        amount,
+        interface,
+        bucketclass_dict,
     ):
         """
         Test bucket creation using the S3 SDK, OC command or MCG CLI.
         The factory checks the bucket's health by default.
         """
-        bucket_factory(amount, interface, bucketclass=bucketclass_dict)
+        buckets = bucket_factory(amount, interface, bucketclass=bucketclass_dict)
+
+        if amount == 3 and interface == "OC":
+            unexpected_log = 'malformed BucketHost "s3.openshift-storage.svc": malformed subdomain name "s3"'
+            rook_op_pod = get_operator_pods()
+            pod_log = get_pod_logs(pod_name=rook_op_pod[0].name)
+            assert not (
+                unexpected_log in pod_log
+            ), f"Bucket notification errors found {unexpected_log}"
+
+        if amount not in [100, 1000] or interface != "CLI":
```

Comment:
Why we skipping the deletion testing in these cases? It looks like the original `test_delete_bucket` did cover them:
```
        ids=[
            "3-S3-DEFAULT-BACKINGSTORE",
            "3-CLI-DEFAULT-BACKINGSTORE",
            "3-OC-DEFAULT-BACKINGSTORE",
            "100-S3-DEFAULT-BACKINGSTORE",
            "100-OC-DEFAULT-BACKINGSTORE",
            "1000-S3-DEFAULT-BACKINGSTORE",
            "1000-OC-DEFAULT-BACKINGSTORE",
            "1-OC-PVPOOL",
            "1-CLI-PVPOOL",
        ],
    )
    def test_bucket_delete(
    ``` 
(From the original test_bucket_deletion.py)

Reply:
we are only skipping for interface CLI and amount is either 100 or 1000 which is not there in the original test